homeassistant/components/xiaomi_miio/humidifier.py

```diff
@@ -185,6 +196,15 @@ def __init__(self, name, device, entry, unique_id, coordinator):
             self._min_humidity = 30
             self._max_humidity = 80
             self._humidity_steps = 100
+        elif self._model in MODELS_HUMIDIFIER_MJJSQ:
+            self._available_modes = [
```

Comment:
This looks like a constant that can be defined already at module level?

Reply:
Do you mean not getting a mode list from the backend library but defining it as a constant?

Reply:
@MartinHjelmare  I agree with @bieniu. The backend is suppling the available modes. Let us not redefine those in the integration constants. This is consistent with the supported existing models.

Reply:
I mean we can do the list comprehension and store as a constant at the module level. Then just use that constant here.